### DISCUSSION TITLE
Flutter Package of the Week -- Animated Text Kit

### DISCUSSION THREAD

**Discussion by aagarwal1012** [aagarwal1012/Animated-Text-Kit#256] [Show and tell]
Kudos guys!! 🎉🎉 Flutter just uploaded the animated_text_kit (Flutter Package of the Week) on Youtube. I can't believe we will get so much recognition with this Flutter package. 

Link to the video: https://youtu.be/foQTKCQqVWk

cc: @awhitford @SirusCodes @hemilpanchiwala

**Comment by SirusCodes** [comment]
Congratulations everyone!
